base/compiler/ssair/inlining.jl

```diff
@@ -441,6 +439,24 @@ function ir_inline_item!(compact::IncrementalCompact, idx::Int, argexprs::Vector
     return_value
 end
 
+function fix_va_argexprs!(compact::IncrementalCompact,
+    argexprs::Vector{Any}, nargs_def::Int, line_idx::Int32)
+    newargexprs = Any[]
+    for i in 1:(nargs_def-1)
+        push!(newargexprs, argexprs[i])
+    end
```

Comment:
why use such an awkward way to slice this array?
```suggestion
    newargexprs = argexprs[1:(nargs_def-1)]
```

Reply:
#43676.